data/gui/themes/default/widget/menu_button_game_actions.cfg

```diff
@@ -0,0 +1,109 @@
+#textdomain wesnoth-lib
+###
+### Definition of the menu_button when used like the Menu/Action buttons of the game screen.
+###
+
+#define _GUI_TEXT FONT_SIZE FONT_COLOR
+	[text]
+                x = {GUI__TEXT_HORIZONTALLY_CENTRED}
```

Comment:
inconsistent indent

Reply:
The original doesn't seem to be wrongly indented.
https://github.com/wesnoth/wesnoth/blob/8863cd1bcd3d4ea55a3f6b99225debcd932b5eec/data/gui/themes/default/widget/menu_button_default.cfg#L6-L18.